packages/amplify-graphql-auth-transformer/src/__tests__/resolvers/subscriptions.test.ts

```diff
@@ -26,27 +26,27 @@ const defaultRoleDefinitions: Record<AuthProvider, Array<RoleDefinition>> = {
       provider: 'iam',
       strategy: 'public',
       static: true,
-      claim: 'testClaim',
+      claim: 'unauthRole',
     },
     {
       provider: 'iam',
       strategy: 'private',
       static: true,
-      claim: 'testClaim',
+      claim: 'authRole',
     },
   ],
   identityPool: [
     {
       provider: 'identityPool',
       strategy: 'public',
       static: true,
-      claim: 'testClaim',
+      claim: 'unauthRole',
```

Comment:
The conflict in this file was on `claim` fields. Resolved in favor of `gen2-release`.